demo/admin/src/products/future/generated/ProductForm.tsx

```diff
@@ -45,7 +48,13 @@ const rootBlocks = {
     image: DamImageBlock,
 };
 
-type FormValues = GQLProductFormDetailsFragment & {
+type FormValues = Omit<GQLProductFormDetailsFragment, "dimensions"> & {
+    useDimensions: boolean;
```

Comment:
I'm not sure about the use prefix because it reminds me of a hook. Could we find something better?

Reply:
it's useAlternativeAddress in the [demo](https://github.com/vivid-planet/comet/pull/2232/files#diff-62e570ff3ed31759613e194bc4fca59606ed16d03e455d24f6b20054c1283957R84). basically it's a boolean, booleans can be predicates, convention for predicates is is-prefix, but isDimensions does not feel right. maybe saveDimensions?

Reply:
Yes, useAlternativeAddress makes sense, since there's also another address field. But here there aren't other dimensions 😁 

Maybe `hasDimensions` or `dimensionsEnabled`?